### PR TITLE
S0014 20230920

### PR DESCRIPTION
**Pull request by AdriJD**
This PR adds an entry for the `mbs_s0014_2030920` map-based noise simulations for the SAT. These are based on the [mbs-noise-sims-sat](https://github.com/simonsobs/pwg-scripts/tree/master/pwg-tds/mbs-noise-sims-sat) time-domain simulations. Also updated the `README.md` and the `LOGBOOK.md` files with the new entry and updated the name of the slack channel in the README.

### CODE REVIEW

File: mbs-s0014-20230920/README.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+* 2023-09-20: first release.
+
+## Summary
+
+300 realistic realizations of the SAT map noise, estimated from the [mbs-noise-sims-sat](https://github.com/simonsobs/pwg-scripts/tree/master/pwg-tds/mbs-noise-sims-sat) time-domain noise simulations. Simulations cover the full SAT footprint and utilize the tiled noise model.
+
+## Noise model
+
+This release is based on the [mbs-noise-sims-sat](https://github.com/simonsobs/pwg-scripts/tree/master/pwg-tds/mbs-noise-sims-sat) time-domain simulations, available on `NERSC` at:
```
Comment (zonca) [inline mbs-s0014-20230920/README.md:15]: it looks like this is a link to the software used for the sims, is there any link (even in the wiki not publicly available) to the documentation of the simulations?

File: mbs-s0014-20230920/README.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+    /global/cfs/cdirs/sobs/sims/sat-noise-sims/v1
+
+These time-domain simulations include detector and atmospheric noise scaled to match a full year of sensitivity split into two disjoint maps for each frequency band. 
+
+Using [`mnms`](https://github.com/simonsobs/mnms), we generate 300 realizations of the noise using map-based methods. Specifically, we use the tiled model, governed by the configured [parameters](parameters/so_sat_v1_f1.yaml).
```
Comment (zonca) [inline mbs-s0014-20230920/README.md:21]: can you tag a release of `mnms` and mention it here?

File: mbs-s0014-20230920/README.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+* A mask has been applied to the map-domain simulations. The masks are stored here: `/global/cfs/cdirs/sobs/v4_sims/mbs/mbs_s0014_20230920/car/masks/coadd_SAT_{bands}_mask_obs_hp.fits`.
+* See these [slides](https://drive.google.com/file/d/1ZkwpjYQVCkmbRk5RWOp0Qq4CPsynBH31/view?usp=sharing) for validation of the map-based simulations.
+
+## Available maps
+
+Maps are available in the HEALPix format. Maps are in Equatorial Coordinates, `K_CMB` units, FITS format.
+
+The maps are available on `NERSC` at:
+
+    /global/cfs/cdirs/sobs/v4_sims/mbs/mbs_s0014_20230920/out/run00
```
Comment (zonca) [inline mbs-s0014-20230920/README.md:34]: it would be interesting to add how much space they occupy

**Review by zonca — COMMENTED** [review]
Nice that all scripts and parameter files are made available in the repository!
Just minor improvements suggested.

File: mbs-s0014-20230920/README.md (hunk 1 above)
Comment (AdriJD) [reply mbs-s0014-20230920/README.md:15]: I can't find any wiki page about these TOD simulations. They were meant as an intermediate product to test the MBS pipeline. I added a list of specifications about the TOD sims to this page to make it clear what went in to the simulations.

File: mbs-s0014-20230920/README.md (hunk 2 above)
Comment (AdriJD) [reply mbs-s0014-20230920/README.md:21]: I have added the tags for `mnms` and `sofind`

File: mbs-s0014-20230920/README.md (hunk 3 above)
Comment (AdriJD) [reply mbs-s0014-20230920/README.md:34]: Added! It's 215 GB

**Comment by AdriJD** [comment]
Thanks! I have added your suggestion.

**Review by zonca — APPROVED** [review]

**Pull request merged**